src/main/java/plugins/nate/smp/managers/PvPManager.java

```diff
@@ -0,0 +1,59 @@
+package plugins.nate.smp.managers;
+
+import org.bukkit.entity.Player;
+import org.bukkit.potion.PotionEffectType;
+
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+import java.util.concurrent.ConcurrentHashMap;
+
+public class PvPManager {
+    private static final long COMBAT_COOLDOWN = 45 * 1000;
```

Comment:
45 seconds is **awhile**. Maybe 15?

```suggestion
    private static final long COMBAT_COOLDOWN = 15 * 1000;
```